### DISCUSSION TITLE
P4 to GitHub Migration - Error while uploading 5GB of data files

### DISCUSSION THREAD

**Discussion by me948** [community/community#29756] [Actions]
I have cloned the files from P4 to local and getting below error while pushing into GitHub. I tried with LFS too and able to push only specific files, but one of my repository having different kind of files (up to 5 GB). Missing history details too if I push the large files separately. Please help with a solution to push the complete repository, which is having 5 GB of data files, along with history.

$ git push origin master -f
Enumerating objects: 12600, done.
Counting objects: 100% (12600/12600), done.
Delta compression using up to 2 threads
Compressing objects: 100% (4766/4766), done.
Writing objects: 100% (12600/12600), 297.92 MiB | 15.23 MiB/s, done.
Total 12600 (delta 6202), reused 12600 (delta 6202), pack-reused 0
remote: Resolving deltas: 100% (6202/6202), done.
remote: error: Trace: 64e53187ea489c0ff8a8647ad97c85d71b9574bcdc2aef200af03a416a954415
remote: error: See http://git.io/iEPt8g for more information.
remote: error: File BR-argon_scc-r06/projects/argon/linux/cheetah/native/upgrade/test/schema8/main_3.0 is 112.00 MB; this exceeds GitHub's file size limit of 100.00 MB
remote: error: GH001: Large files detected. You may want to try Git Large File Storage - https://git-lfs.github.com.

$ git push origin master
Enumerating objects: 6098, done.
Counting objects: 100% (6098/6098), done.
Delta compression using up to 2 threads
Compressing objects: 100% (2949/2949), done.
remote: fatal: pack exceeds maximum allowed size (2.00 GiB)
fatal: sha1 file '<stdout>' write error: Broken pipe
error: remote unpack failed: index-pack abnormal exit

**Comment by airtower-luna** [comment]
The error tells you exactly where the problem is:

> remote: error: File BR-argon_scc-r06/projects/argon/linux/cheetah/native/upgrade/test/schema8/main_3.0 is 112.00 MB; this exceeds GitHub's file size limit of 100.00 MB

That's a hard limit, check https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github for the alternatives you have (with GitHub it's mostly LFS).
